# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/test/mochitest/browser.ini
# Commit: 5791073c28b7
# Full Hash: 5791073c28b727964776360a5b4dfd5c004ce152
# Author: Hiroyuki Ikezoe <hikezoe.birchill@mozilla.com>
# Date: 2021-04-13 03:56:47
# Description:
#   Bug 1703762 - Delegate StartAsyncAutoscroll and StopAsyncAutoscroll calls to the proper widget mPopupContentView. r=botond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D111575
# ==============================================================================

diff -r 4a25e6b0bf09 -r 5791073c28b7 gfx/layers/apz/test/mochitest/browser.ini
--- a/gfx/layers/apz/test/mochitest/browser.ini	Mon Apr 12 22:50:22 2021 +0000
+++ b/gfx/layers/apz/test/mochitest/browser.ini	Mon Apr 12 23:05:40 2021 +0000
@@ -6,6 +6,8 @@
   apz_test_utils.js
   helper_browser_test_utils.js
   !/browser/base/content/test/forms/head.js
+  !/browser/components/extensions/test/browser/head.js
+  !/browser/components/extensions/test/browser/head_browserAction.js
 
 [browser_test_group_fission.js]
 skip-if = (os == 'win' && bits == 32) # Some subtests fail intermittently on Win7.
@@ -31,18 +33,10 @@
 support-files =
   helper_test_reset_scaling_zoom.html
 [browser_test_scrollbar_in_extension_popup_window.js]
-support-files =
-  !/browser/components/extensions/test/browser/head.js
-  !/browser/components/extensions/test/browser/head_browserAction.js
 [browser_test_scrolling_in_extension_popup_window.js]
-support-files =
-  !/browser/components/extensions/test/browser/head.js
-  !/browser/components/extensions/test/browser/head_browserAction.js
 [browser_test_scrolling_on_inactive_scroller_in_extension_popup_window.js]
 run-if = (os == 'mac') # bug 1700805
-support-files =
-  !/browser/components/extensions/test/browser/head.js
-  !/browser/components/extensions/test/browser/head_browserAction.js
 [browser_test_scroll_thumb_dragging.js]
 support-files =
   helper_scroll_thumb_dragging.html
+[browser_test_autoscrolling_in_extension_popup_window.js]